ntrans_dataprep.py

```diff
@@ -7,8 +7,10 @@
 
 Unzip BNC to the working directory, and rename the folder `BNC`.
 
-NOTE: The BNC contains around 112.000.000 words (or 5.397.000 sentences).
-It can take around 2 hours to run this script.
+NOTE: The BNC contains around 112.000.000 words (or 6.020.000+ sentences).
+It can take around 1,5 hours to run this script.
+
+The script generates 236.937.946 N-grams.
```

Comment:
Akuli's annoying style nits are back. In programming context:
- using `.` as a thousands separator is confusing, since it's usually the decimal point
- detailed numbers get outdated easily
- `k` for 1000, `M` for 1000000 etc are widely understood

I would use something like `236M N-grams`.